va/va.h

```diff
@@ -4929,6 +4939,47 @@ typedef enum
 #define VA_RENDER_DEVICE_LOCAL      1
 #define VA_RENDER_DEVICE_EXTERNAL   2
 
+/**\brief sub device info
+ * Sub-device is the concept basing on the "device" behind "vaDisplay".
+ * If a device could be divided to several sub device, the task of
+ * decode/encode/vpp could be assigned on one sub-device. So, application
+ * could choose the sub device and attach it to vaDisplay before
```

Comment:
I don't quite like "attach". It sounds like in the future we might be able to attach multiple sub-devices to vaDisplay. Is that we want to have? This will require API change however since since right now we have only `current_sub_device`, but we will need a mask.